doc/api.yml

```diff
@@ -1400,6 +1640,16 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/StudentExamGrade'
+    CreatePointing:
+      type: object
+      properties:
+        pointing_type:
```

Comment:
Rename correctly.

Reply:
checked, change to CreateAttendanceMovement